src/test/java/ca/wolfram/beta/symath/MathUtilsTest.java

```diff
@@ -0,0 +1,21 @@
+package ca.wolfram.beta.symath;
```

Comment:
Not sure about how you do your unit tests, but i usually split multiply assert into different tests. So the test that fails can tell you exactly what failed, instead of having to look at the test debug message